cmd/helm/deployment.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/codegangsta/cli"
+)
+
+func init() {
+	addCommands(deploymentCommands())
+}
+
+func deploymentCommands() cli.Command {
+	return cli.Command{
+		// Names following form prescribed here: http://is.gd/QUSEOF
+		Name:  "deployment",
+		Usage: "Perform deployment-centered operations.",
+		Subcommands: []cli.Command{
+			{
+				Name:      "config",
+				Usage:     "Dump the configuration file for this deployment.",
+				ArgsUsage: "DEPLOYMENT",
+			},
+			{
+				Name:      "manifest",
+				Usage:     "Dump the Kubernetes manifest file for this deployment.",
```

Comment:
there can be multiple manifests, so should this be Dump the latest? or something like that.